### ISSUE TITLE
WhatsApp Integration

### ISSUE THREAD

**Issue by dvargas92495** [dvargas92495/roam-js-extensions#609]
I want to be able to trigger an action which pulls the last N convos along with the last M messages in each convo, which dates attached to each message. 

Then I can easily use a roam query to show me messages which are old than X days old and haven't been replied to. Bonus points if I can write the reply right in roam then press a send button to send it off. 

Essentially, I'm looking to use Roam as a whatsapp client which will have similar features to gmail (reminders to follow up, or reply to a message I missed, etc). 

In Summary:

- Querying data
- Sending messages

**Comment by dvargas92495** [comment]
Audit: Won't do
